additional.yml

```diff
@@ -0,0 +1,14 @@
+flows:
```

Comment:
Perhaps this belongs alongside the tests instead of here?

Reply:
Yeah, I suggest adding it inline in the tests and using `tmp_path` to write the contents to disk during the test. If that's not workable, then moving it to the same subdirectory as the tests is fine.